README.md

```diff
@@ -1,74 +1,71 @@
-[![Build Status](https://travis-ci.com/ecamp/ecamp3.svg?branch=devel)](https://travis-ci.com/ecamp/ecamp3)
-[![Coverage Status](https://coveralls.io/repos/github/ecamp/ecamp3/badge.svg?branch=devel)](https://coveralls.io/github/ecamp/ecamp3?branch=devel)
+<p align="center"><img width="320px" align="center" style="width:320px" src="https://raw.githubusercontent.com/ecamp/ecamp3/devel/logo.svg"/></p><br/>
```

Comment:
```suggestion
<p align="center"><img width="320px" align="center" style="width:320px" src="https://raw.githubusercontent.com/ecamp/ecamp3/devel/docs/logo.svg"/></p><br/>
```

Reply:
Do you want me to move the logo image file to the docs folder that may be removed later in #665 ? Or should I also ignore this change proposal?